crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -2576,10 +2636,23 @@ input TransactionBlockFilter {
 	transactionIds: [String!]
 }
 
+"""
+The kind of transaction blocks that exist.
```

Comment:
maybe something like, other than ProgrammableTransactionBlock, the other kinds are all system transactions?